some-trouble-with-border-in-tests

BIr3791 opened the pull request:
not all tests passed

stanislav-topikha reviewed (changes requested):
add test report and demo link to the description

BIr3791 commented:
local all tests passed...

BIr3791 commented:
https://github.com/mate-academy/layout_search-bar-airbnb/actions/runs/3343487590/jobs/5536755725

https://bir3791.github.io/layout_search-bar-airbnb/

BIr3791 commented:
local tests good but on git test have failed

Bir3791 commented:
- [DEMO LINK](https://Bir3791.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://Bir3791.github.io/layout_search-bar-airbnb/report/html_report/)

DarkMistyRoom commented on src/index.html:
```diff
@@ -11,6 +11,44 @@
+      data-qa="small"
+    >
+      <span class="input_icon-small">
+        <img src="../src/images/Search.svg" alt="finder-logo"
+          width="11"
+          height="11"
+        >
```
```suggestion
        <img
          src="../src/images/Search.svg"
          alt="finder-logo"
          width="11"
          height="11"
        >
```

DarkMistyRoom replied:
Set width and height in the CSS, it is more clear and predictably

Bir3791 replied:
done

DarkMistyRoom commented on src/index.html:
```diff
@@ -11,6 +11,44 @@
+      method="post"
+      data-qa="big"
+    >
+      <span class="input_icon">
+        <img src="../src/images/Search.svg" alt="finder-logo">
+      </span>
```
same for the small one

Bir3791 replied:
done

DarkMistyRoom commented on src/index.html:
```diff
@@ -11,6 +11,44 @@
+      <span class="input_icon">
+        <img src="../src/images/Search.svg" alt="finder-logo">
+      </span>
+      <label>
```
same for the small one

Bir3791 replied:
if i deleted label linter show eror

DarkMistyRoom commented on src/style.css:
```diff
@@ -0,0 +1,106 @@
+  box-sizing: border-box;
+}
+
+body,
+html {
+  margin: 0;
+  padding: 0 4px;
+}
```
```suggestion
body {
  margin: 0;
}
```

DarkMistyRoom replied:
Make it work properly

Bir3791 replied:
done

DarkMistyRoom commented on src/style.css:
```diff
@@ -0,0 +1,106 @@
+  width: 100%;
+  vertical-align: middle;
+  padding-left: 62px;
+  font-family: "Avenir", sans-serif;
```
Move it to the html selector and inherit it where the need

Bir3791 replied:
done

DarkMistyRoom commented on src/style.css:
```diff
@@ -0,0 +1,106 @@
+
+}
+
+.serch_bar-condition input {
```
Don't increase a specificity, use class instead

DarkMistyRoom commented on src/style.css:
```diff
@@ -0,0 +1,106 @@
+}
+
+.input_icon-small {
+  position: absolute;
```
Don't repeat styles, use modifiers. Check it in the checklist

DarkMistyRoom reviewed (changes requested):
Proofread the [CHECKLIST](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)

Bir3791 commented:
I had too many mistakes in the styles, so I grouped them, removed the redundant ones and merged the duplicates. added variables and fixed critical bugs.
BUT git find some error, i don't see it local.

- [DEMO LINK](https://Bir3791.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://Bir3791.github.io/layout_search-bar-airbnb/report/html_report/)

alexandra-protyanova commented on src/index.html:
```diff
@@ -11,6 +11,45 @@
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="serch_bar-container"
+      action="#"
+      method="post"
+      data-qa="big"
+    >
```
```suggestion
    <form 
      class="serch_bar-container"
      action="#"
      method="post"
      data-qa="big"
    >
```

Bir3791 replied:
fixed

alexandra-protyanova commented on src/index.html:
```diff
@@ -11,6 +11,45 @@
+        class="input_icon"
+      >
+
+      <input
+      id="input_big"
+      type="text"
+      class="input_big"
+      placeholder="Try &#34Los Angeles&#34"
+      >
```
```suggestion
      <input
        id="input_big"
        type="text"
        class="input_big"
        placeholder="Try &#34Los Angeles&#34"
      >
```

alexandra-protyanova replied:
read checklist about code style

Bir3791 replied:
done

alexandra-protyanova commented on src/style.css:
```diff
@@ -0,0 +1,108 @@
+/* #endregion  */
+
+/* #region inputs and logos */
+.input_big,
```
create common class for inputs and move all repetitive styles there

Bir3791 replied:
fixed

Bir3791 replied:
fixed

Bir3791 commented:
added modificators.

- [DEMO LINK](https://Bir3791.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://Bir3791.github.io/layout_search-bar-airbnb/report/html_report/)
